roadmap/implementers-guide/src/node/disputes/dispute-coordinator.md

```diff
@@ -199,11 +202,11 @@ time participation is faster than approval, a node would do double work.
 ### Ensuring Chain Import
 
 While in the previous section we discussed means for nodes to ensure relevant
-votes are recorded so attackers get slashed properly, it is crucial to also
-discuss the actual chain import. Only if we guarantee that recorded votes will
-also get imported on chain (on all potential chains really) we will succeed in
-executing slashes. Again approval votes prove to be our weak spot here, but also
-backing votes might get missed.
+votes are recorded so lazy approval checkers get slashed properly, it is crucial 
+to also discuss the actual chain import. Only if we guarantee that recorded votes 
+will also get imported on chain (on all potential chains really) we will succeed 
+in executing slashes. Again approval votes prove to be our weak spot here, but 
+also backing votes might get missed.
```

Comment:
Nice @BradleyOlson64 ! Thanks! One thing to maybe highlight some more: As the only reason why we want to slash approval checkers is to give a strong incentive to actually do the checking, we just have to make it likely for them to get caught. We don't have to assume that they are willing to go through sophisticated attacks to avoid their vote to be recorded on some edge cases, as going through this trouble is not sensible: You are not lazy if you are going through a lot of trouble - it defeats the point. Paying engineers and infrastructure, devops, maintaining a fork, .. to pull off such an attack will never be cheaper than just doing the validation.

Reply:
Welcome! I think I've addressed your concern now, but feel free to make sure.